src/__tests__/01b-create-users.ts

```diff
@@ -134,9 +137,32 @@ describe("username tests", () => {
     await expect(userWallet2.setUsername({ username })).rejects.toThrow()
   })
 
-  // FIXME: failing for some reason
-  // it('sets account status correctly', async () => {
-  //   await AdminOps.setAccountStatus({uid: userWallet2._id, status: 'locked'})
-  //   await expect(userWallet2.status).toBe('locked')
-  // })
+  it("sets account status correctly", async () => {
+    await AdminOps.setAccountStatus({ uid: userWallet2.user._id, status: "locked" })
+    userWallet2 = await getUserWallet(2)
+    await expect(userWallet2.user.status).toBe("locked")
+  })
+
+  it("set 2fa for user0", async () => {
+    const { secret } = userWallet0.generate2fa()
+    const token = generateToken(secret)!.token
```

Comment:
can you fix the typescript warning?

Reply:
I'm not sure how to do that. This is a part of the node-2fa .d.ts file - 
```
export declare function generateToken(secret: string): {
    token: string;
} | null;
```

As you can see the return value for generateToken could be null. We can probably override this by declaring our own file but that doesn't seem to be an improvement over the non-null assertion operator we're using currently.

Reply:
> @krtk6160 you may have taken one of my commit from main that I mistakenly pushed. you may want to rebase (to remove the `"noImplicitAny": true,`)

Yep, thanks.